olive/passes/onnx/optimum_merging.py

```diff
@@ -74,14 +72,4 @@ def new_byte_size_func(_):
         # onnx.save will fail if the directory doesn't already exist
         output_model_path = resolve_onnx_path(output_model_path, "decoder_model_merged.onnx")
 
-        olive_model = model_proto_to_olive_model(merged_model, output_model_path, config)
```

Comment:
Since accelerator spec is not used anymore, could you remove the `is_accelerator_agnostic` method at line 24?

Reply:
I reverted changes to this file, since they weren't meant to be committed.